Assets/LeapMotion/Core/Scripts/Utils/HandUtils.cs

```diff
@@ -327,4 +331,151 @@ public class HandEvent : UnityEvent<Hand> { }
 
   }
 
+  /// <summary>
+  /// Utility methods for constructing and manipulating Leap hand object data.
+  /// </summary>
+  public static class HandUtils {
+
+    /// <summary>
+    /// Fills the Hand object with the provided hand data. You can pass null for the
+    /// fingers input; this will leave the hand's finger data unmodified.
+    /// </summary>
+    public static void Fill(this Hand toFill,
+                            long frameID,
+                            int id,
+                            float confidence,
+                            float grabStrength,
+                            float grabAngle,
+                            float pinchStrength,
+                            float pinchDistance,
+                            float palmWidth,
+                            bool isLeft,
+                            float timeVisible,
+                            /* Arm arm,*/
```

Comment:
Hm why does this not contain an initialization for the Arm?

Reply:
ask @jselstad

Reply:
too bad @jselstad didn't add a comment explaining it
@jselstad @jselstad @jselstad